terracotta/server/rgb.py

```diff
@@ -29,25 +30,43 @@ class Meta:
     g = fields.String(required=True, description="Key value for green band")
     b = fields.String(required=True, description="Key value for blue band")
     r_range = fields.List(
-        fields.Number(allow_none=True),
+        StringOrNumber(allow_none=True, validate=validate_stretch_range),
         validate=validate.Length(equal=2),
         example="[0,1]",
         missing=None,
-        description="Stretch range [min, max] to use for red band as JSON array",
+        description=(
+            "Stretch range [min, max] to use for the red band as JSON array. "
+            "Min and max may be numbers to use as absolute range, or strings "
+            "of the format p<int> with an integer between 0 and 100 to use "
```

Comment:
Don't use `<>`, angled brackets get parsed as HTML in the API docs. Either escape with `&lt;` and `&gt;` or use a different character.